client/app/vps/monitoring/vps-monitoring.constant.js

```diff
@@ -0,0 +1,75 @@
+angular.module("managerApp").constant("VpsMonitoringConstant", {
+	colors : ['#F1C40F', '#3498DB', '#717984', '#72C02C'],
+    option1 : {
```

Comment:
this is bad for so many reasons, please indent correctly by the way

Reply:
I must be a way to declare those colors as constants and avoid so much code for chartjs integration and there's still error in the formating. Please use a plugin. line 21, 37, 60, also double quotes...